packages/apps/docs/.changeset/lucky-teachers-roll.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
You should be able to delete this file and the `.changeset` folder in the path /packages/apps/docs.  The `.changeset` folder should only exist in the root of the repository